qutebrowser/misc/tabsfinder.py

```diff
@@ -0,0 +1,43 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+import re
+from collections import namedtuple
+
+from qutebrowser.utils import objreg
+
+Tab = namedtuple('Tab', 'win_id index url title')
```

Comment:
Namedtuples are kind of confusing because they can also be used as a tuple even if they aren't.

Please use the [attrs](http://www.attrs.org/en/stable/) module instead (search for `@attr.s` in qutebrowser's codebase for some examples).

Reply:
Nice suggestion. Done.